projectile.el

```
@@ -975,6 +999,12 @@ they are excluded from the results of this function."
            (projectile-get-sub-projects-files)))
    (t (projectile-files-via-ext-command (projectile-get-ext-command)))))
 
+(defun projectile-get-repo-ignored-files ()
+  "Get a list of the files ignored in the project"
+  (let ((cmd (projectile-get-ext-ignored-command)))
```

There's also `when-let`. Or you can just raise an error.